packages/terra-dropdown-button/tests/wdio/split-button-spec.js

```diff
@@ -207,7 +207,7 @@ Terra.describeViewports('Split Button', ['medium'], () => {
     });
 
     it('opens the long option dropdown', () => {
-      browser.keys(['Escape']);
+      browser.refresh();
```

Comment:
Why is this needed?

Reply:
not needed reverted to [previous code](https://github.com/cerner/terra-core/blob/3d4cbdca1a5c96ae211cdcf5bdb5847ecfdc965c/packages/terra-dropdown-button/tests/wdio/split-button-spec.js#L210)